pipelines/tekton/aiedge-e2e/tasks/kserve-download-model.yaml

```diff
@@ -32,3 +35,7 @@ spec:
     name: workspace
   - description: The S3 secret.
     name: s3-secret
+  - description: |
```

Comment:
We'll have to set this as an `optional: true` workspace to support the workflows where a custom cert is not required for the Pipeline to run successfully and update the script to populate the env vars only when this workspace is mounted and the appropriate file exists

Reply:
Yes, these were just while it was a draft and I was testing. Using self signed certs is now optional for both git and s3